### PR TITLE
Fix ini_get() for boolean values

### PR DESCRIPTION
**Pull request by deguif**
| Q             | A
| ------------- | ---
| Branch?       | 2.8
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

Currently setting `false` or `off`, ... value to configure some PHP ini directives will make this evaluated to `true` as this is equal to a non empty string.

### CODE REVIEW

**Review by ndench — APPROVED** [review]
LGTM :+1:

File: src/Symfony/Component/ClassLoader/Tests/ApcClassLoaderTest.php [hunk 1]
```diff
@@ -19,7 +19,7 @@ class ApcClassLoaderTest extends TestCase
 {
     protected function setUp()
     {
-        if (!(ini_get('apc.enabled') && ini_get('apc.enable_cli'))) {
+        if (!(\filter_var(ini_get('apc.enabled'), \FILTER_VALIDATE_BOOLEAN) && \filter_var(ini_get('apc.enable_cli'), \FILTER_VALIDATE_BOOLEAN))) {
```
Comment (nicolas-grekas) [inline src/Symfony/Component/ClassLoader/Tests/ApcClassLoaderTest.php]: Can you please remove all the `\` added in front of `filter_var()` and `FILTER_VALIDATE_BOOLEAN` in the PR?
We don't add these except for a specific short list of functions.

Comment (deguif) [reply src/Symfony/Component/ClassLoader/Tests/ApcClassLoaderTest.php]: Ok, changed ;)

**Review by nicolas-grekas — APPROVED** [review]

**Comment by ro0NL** [comment]
See also https://github.com/symfony/symfony/blob/2cd1e118478d0a57ff8c43ebdd8d6a54c40f5805/src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php#L86-L87

https://github.com/symfony/symfony/blob/2cd1e118478d0a57ff8c43ebdd8d6a54c40f5805/src/Symfony/Component/Debug/Debug.php#L48 also maybe

**Comment by ro0NL** [comment]
oh wait.. i checked master :) AboutCommand would be for 3.4

**Comment by deguif** [comment]
Yes, will do another PR which will target 3.4 after ;)

**Comment by deguif** [comment]
I'm also going to check every ini_get, I just focused on `opcache` and `apc` for the moment, going to send them ASAP

**Comment by deguif** [comment]
Ok, just added
- `log_errors`
- `xcache.cacher`
- `wincache.ocenabled`

Not sure about `eaccelerator.enable` as it doesn't use the macro `STD_PHP_INI_BOOLEAN`

**Review by ogizanagi — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @deguif.

**Pull request merged**